examples/ecs/system_graph.rs

```diff
@@ -0,0 +1,53 @@
+use bevy::prelude::*;
+
+fn main() {
+    // SystemGraphs can be used to specify explicit system dependencies without specifying
+    // explicit labels for each system.
+    //
+    // Labels can be used alongside SystemGraphs to help specify dependency relationships
+    // between graphs, betwen standalone systems, or between crate boundaries.
+
+    // These three systems will run sequentially one after another.
+    let sequential = SystemGraph::new();
+    sequential
+        .root(print_system("Sequential 1"))
+        .then(print_system("Sequential 2"))
+        .then(
+            print_system("Sequential 3")
+                .system()
+                .label("Sequential End"),
+        );
+
+    // Graphs nodes can fork into multiple dependencies.
+    let wide = SystemGraph::new();
+    let (mid_1, mid_2, mid_3) = wide
+        .root(print_system("Wide Start").system().after("Sequential End"))
```

Comment:
```suggestion
        .root(print_system("Wide Start").after("Sequential End"))
```
Omitting `.system()` should be okay here now.